cornac/models/recvae/recom_recvae.py

```diff
@@ -0,0 +1,222 @@
+# Copyright 2018 The Cornac Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+
+import numpy as np
+
+from ..recommender import Recommender
+from ...exception import ScoreException
+from copy import deepcopy
+
+
+from tqdm.auto import trange
+class RecVAE(Recommender):
+    def __init__(
```

Comment:
Please update docstrings of this class and add more details on the usage of parameters. For example, you can refer to any model, e.g., https://github.com/PreferredAI/cornac/blob/b39f4159ddce32f759cc6b77c7258b2d722aced0/cornac/models/mf/recom_mf.py#L30

Reply:
Added docstring for main class